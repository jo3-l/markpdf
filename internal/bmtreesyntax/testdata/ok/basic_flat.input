bookmark one, 1
bookmark two, 2
bookmark three, 3