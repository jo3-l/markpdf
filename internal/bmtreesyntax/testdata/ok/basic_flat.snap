- "bookmark one" @ p1
- "bookmark two" @ p2
- "bookmark three" @ p3